pisound-btn/pisound-btn.c

```diff
@@ -61,14 +61,10 @@ enum action_e
 static const char *const DOWN_VALUE_NAME           = "DOWN";
 static const char *const UP_VALUE_NAME             = "UP";
 
-static const char *const CLICK_1_VALUE_NAME        = "CLICK_1";
-static const char *const CLICK_2_VALUE_NAME        = "CLICK_2";
-static const char *const CLICK_3_VALUE_NAME        = "CLICK_3";
+static const char *CLICK_NAME="CLICK_%u";
```

Comment:
Please prefer using the same type for global constants as in the other lines - the additional const makes this pointer non-assignable.